examples/example_tissuemask.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>You will need to change this to pip install tiatoolbox once ready to merge.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I have it noted ;)</p>